pytorch_lightning/callbacks/finetuning.py

```diff
@@ -83,6 +83,7 @@ def finetune_function(self, pl_module, current_epoch, optimizer, optimizer_idx):
 
     def __init__(self):
         self._internal_state: Dict[int, List[Dict[str, Any]]] = {}
```

Comment:
this is a bit confusing can we rather call it `_internal_optimizer_state` as it it related to optimizer not callback

Reply:
Done !